plexus-compiler-test/src/main/java/org/codehaus/plexus/compiler/AbstractCompilerTest.java

```diff
@@ -72,34 +71,25 @@ public abstract class AbstractCompilerTest {
     private Map<String, Compiler> compilers;
 
     @Inject
-    private ArtifactRepositoryLayout repositoryLayout;
+    private LocalRepositoryProvider localRepositoryProvider;
```

Comment:
Why not lrm only?

Reply:
Oh i might missed the point here, nvm

Reply:
`lrm` - `LocalRepositoryManager`  is not a `bean`  as I know.

Reply:
yes, I meant repo sys session.lrm but now I see you actually want to create lrm on custom place, ok